functions/utils/nodemailer/connectMailer.ts

```diff
@@ -0,0 +1,33 @@
+import * as nodemailer from "nodemailer";
+
+export default async function connectMailer(): nodemailer.Transporter {
+  return nodemailer.createTransport({
+    host: process.env.NODEMAILER_HOST ?? "",
+    port: process.env.NODEMAILER_PORT ?? "",
+    secure: true,
+    auth: {
+      user: process.env.NODEMAILER_USERNAME ?? "",
+      pass: process.env.NODEMAILER_PASSWORD ?? "",
+    },
+    tls: {
+      rejectUnauthorized: false,
+    },
+  });
```

Comment:
Local typing error for the input type. You can fix this by asserting that the input is of type `nodemailer.SendMailOptions`